components/tab/IconTab.js

```diff
@@ -0,0 +1,65 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import Box from '../box';
+import cx from 'classnames';
+import theme from './theme.css';
+
+import {
+  IconCalendarMediumOutline,
+  IconCheckmarkMediumOutline,
+  IconContactsMediumOutline,
+  IconInvoiceMediumOutline,
+  IconProductsMediumOutline,
+  IconStatsMediumOutline,
+  IconTimerMediumOutline,
+} from '@teamleader/ui-icons';
+
+const iconMap = {
+  crm: IconContactsMediumOutline,
+  invoices: IconInvoiceMediumOutline,
+  products: IconProductsMediumOutline,
+  stats: IconStatsMediumOutline,
+  time: IconTimerMediumOutline,
+  deals: IconCheckmarkMediumOutline,
+  planning: IconCalendarMediumOutline,
+};
```

Comment:
We do not want any specific icons here. It should be able to accept any icon as a prop; otherwise if we want to update project A with icon B, we'd have to `teamleader-ui`. Not scalable!

Reply:
I thought it was said there was no other way to solve this? I asked this before that I wanted to only import the icon that's being requested, but that didn't work. I assumed a huge iconMap was the only way out. I don't know how to do it differently -> tip?

Reply:
Like I said, we should import the icon that we want in the `IconTab` in the project that needs it. eg in `core`

Reply:
This is idd a way better approach. 👍 

 :ok: by @InstaK on 02/01/2018 15:37:03